internal/server/server.go

```diff
@@ -131,15 +132,21 @@ type errorVars struct {
 
 type ErrorHandlerClient interface {
 	MyPermissions(sirius.Context) (sirius.PermissionSet, error)
+	GetDeputyDetails(sirius.Context, int, int) (sirius.DeputyDetails, error)
 }
 
-func errorHandler(logger Logger, client ErrorHandlerClient, tmplError Template, prefix, siriusURL string) func(next Handler) http.Handler {
+func errorHandler(logger Logger, client ErrorHandlerClient, tmplError Template, prefix, siriusURL string, defaultPATeam int) func(next Handler) http.Handler {
 	return func(next Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			myPermissions, err := client.MyPermissions(getContext(r))
 
```

Comment:
We want to handle errors when they happen. With how it is currently, if getting the permissions fails, it will still make a call to try and get the deputy details.